config/locales/pt-BR.yml

```diff
@@ -0,0 +1,265 @@
+pt-BR:
```

Comment:
This is fine for now, but you could split into smaller files just like we have into app folder.

models/
views/

for specific files

and for the rails default translations you could add them into a `default.pt-br.yml`